index.js

```diff
@@ -2,32 +2,42 @@
 
 /* eslint-disable no-underscore-dangle */
 class NotificationBase {
-
     /**
      * Constructor for Build Notification
      * @method constructor
      * @param {Object}      config              Configuration specific to the plugin
-     * @param {Object}      server              API server
-     * @param {String}      eventName           Event name the notification plugin listen to
      * @return {NotificationBase}
      */
-    constructor(config, server, eventName) {
+    constructor(config) {
         this.config = config;
-        this.server = server;
-        this.eventName = eventName;
+    }
+
+    /**
+     * Get event list for receiving Screwdriver API events
+     * @method events
```

Comment:
This should be a getter, so it shouldn't be a method call. This means that getting the events would look something like this:

```
const myEvents = myNotificationObject.events;

console.log(myEvents); // prints the array of events
```